src/test/test-exec-util.c

```diff
@@ -402,7 +402,7 @@ TEST(error_catching) {
         if (access(name, X_OK) < 0 && ERRNO_IS_PRIVILEGE(errno))
                 return;
 
-        r = execute_directories(dirs, DEFAULT_TIMEOUT_USEC, NULL, NULL, NULL, NULL, EXEC_DIR_NONE);
+        r = execute_directories(dirs, DEFAULT_TIMEOUT_USEC, NULL, NULL, NULL, NULL, 0);
```

Comment:
maybe insert `/* flags= */` comment though?

(and maybe for the 4 NULL args too?)